modules/core/depinject.go

```diff
@@ -36,9 +40,9 @@ type ModuleInputs struct {
 	Cdc    codec.Codec
 	Key    *storetypes.KVStoreKey
 
-	ConsensusHost clienttypes.ConsensusHost
```

Comment:
Unfortunately I haven't figured out a way to inject this interface type into the DI container yet. This is not a module output from another module and is instead created in `app.go` (v1). The concrete implementation type is `ibc tendermint` in almost all cases for cometbft chains. The constructor takes the staking keeper as an argument.

app v1 snippet:
```
consensusHost := ibctm.NewConsensusHost(app.StakingKeeper)
app.IBCKeeper = ibckeeper.NewKeeper(...args, consensusHost)
```

Reply:
ConsensusHost will be deleted by the time this is picked back up